lib/eblox/data/post.ex

```diff
@@ -48,8 +49,10 @@ defmodule Eblox.Data.Post do
 
   defmodule Walker do
     @moduledoc false
-    def prewalk({:a, %{class: "tag"}, [text]} = elem, acc),
-      do: {elem, Map.update(acc, :tags, [text], &Enum.uniq([text | &1]))}
+    def prewalk({:a, %{class: "tag"}, [text]} = elem, acc) do
+      tag = String.downcase(text)
```

Comment:
Nah, I finally got the root of all the misunderstanding :) We need to preserve the downcased tag as `data-` attribute to avoid inconsistency.
```suggestion
    def prewalk({:a, %{class: "tag", "data-tag": tag}, [text]} = elem, acc) do
```
See above.